tokio-executor/src/threadpool/park/boxed.rs

```diff
@@ -29,17 +29,17 @@ where
     fn park(&mut self) -> Result<(), Self::Error> {
         self.0.park().map_err(|e| {
             warn!(
-                "calling `park` on worker thread errored -- shutting down thread: {}",
-                e
+                message = "calling `park` on worker thread errored -- shutting down thread",
+                error = %e
```

Comment:
What is the `%` syntax?

Reply:
`%` is shorthand that indicates the field should be recorded using its `fmt::Display` implementation. See https://docs.rs/tracing/0.1.5/tracing/macro.span.html#recording-fields

Reply:
(when https://github.com/tokio-rs/tracing/pull/277 is released, we can just record the error using its `std::error::Error` implementation)